test/test-commit-message.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+travis_regex='^[a-zA-Z0-9]\+: '
```

Comment:
Prefix should be only lower case, and allow ', '  (comma space)
eg:

resources: file: Blah blah

or

pgraph, lib: Do thing